geode-apis-compatible-with-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/string/AbstractBitOpIntegrationTest.java

```diff
@@ -53,64 +62,65 @@ public void bitOp_errors_givenTooFewArguments() {
   @Test
   public void bitop_givenInvalidOperationType_returnsSyntaxError() {
     assertThatThrownBy(
-        () -> jedis.sendCommand(Protocol.Command.BITOP, "invalidOp", "destKey", "srcKey"))
-            .hasMessageContaining(ERROR_SYNTAX);
+        () -> jedis.sendCommand(hashTag, Protocol.Command.BITOP, "invalidOp", "destKey" + hashTag,
+            "srcKey" + hashTag)).hasMessageContaining(ERROR_SYNTAX);
   }
 
   @Test
   public void bitop_givenSetFails() {
-    jedis.sadd("foo", "m1");
-    assertThatThrownBy(() -> jedis.bitop(BitOP.AND, "key", "foo"))
+    jedis.sadd(srcKey, "m1");
+    assertThatThrownBy(() -> jedis.bitop(BitOP.AND, destKey, srcKey))
         .hasMessageContaining(ERROR_WRONG_TYPE);
-    assertThatThrownBy(() -> jedis.bitop(BitOP.OR, "key", "foo"))
+    assertThatThrownBy(() -> jedis.bitop(BitOP.OR, destKey, srcKey))
         .hasMessageContaining(ERROR_WRONG_TYPE);
-    assertThatThrownBy(() -> jedis.bitop(BitOP.XOR, "key", "foo"))
+    assertThatThrownBy(() -> jedis.bitop(BitOP.XOR, destKey, srcKey))
         .hasMessageContaining(ERROR_WRONG_TYPE);
-    assertThatThrownBy(() -> jedis.bitop(BitOP.NOT, "key", "foo"))
+    assertThatThrownBy(() -> jedis.bitop(BitOP.NOT, destKey, srcKey))
         .hasMessageContaining(ERROR_WRONG_TYPE);
   }
 
   @Test
   public void bitopNOT_givenMoreThanOneSourceKey_returnsError() {
     assertThatThrownBy(
-        () -> jedis.sendCommand(Protocol.Command.BITOP, "NOT", "destKey", "srcKey", "srcKey2"))
-            .hasMessageContaining(ERROR_BITOP_NOT);
+        () -> jedis.sendCommand(
+            hashTag, Protocol.Command.BITOP, "NOT", destKey, srcKey, "srcKey2" + hashTag))
+                .hasMessageContaining(ERROR_BITOP_NOT);
   }
 
   @Test
   public void bitopNOT_givenNothingLeavesKeyUnset() {
-    assertThat(jedis.bitop(BitOP.NOT, "key", "foo")).isEqualTo(0);
-    assertThat(jedis.exists("key")).isFalse();
+    assertThat(jedis.bitop(BitOP.NOT, destKey, srcKey)).isEqualTo(0);
+    assertThat(jedis.exists(destKey)).isFalse();
   }
 
   @Test
   public void bitopNOT_givenNothingDeletesKey() {
-    jedis.set("key", "value");
-    assertThat(jedis.bitop(BitOP.NOT, "key", "foo")).isEqualTo(0);
-    assertThat(jedis.exists("key")).isFalse();
+    jedis.set(destKey, "value");
```

Comment:
if you're making other changes, "value" could be extracted. it's used in a few places